2025-10-03 17:54:25,410 [Logger](CRITICAL): Uncaught exception
Traceback (most recent call last):
  File "backseat_app/main.py", line 22, in <module>
    from backseat_app.processing import MarlProcessor
  File "/home/maiv/lrauv_sim/lrauv-backseat-marl/backseat_app/processing.py", line 5, in <module>
    from backseat_app.utils import TargetTracking
  File "/home/maiv/lrauv_sim/lrauv-backseat-marl/backseat_app/utils.py", line 13, in <module>
    import utm
ModuleNotFoundError: No module named 'utm'
